src/utils.js

```diff
@@ -0,0 +1,152 @@
+import Todo from './todo.js';
+
+export default class Methods {
+static setLocalStorageData = (todo) => {
+  const item = JSON.stringify(todo);
+  localStorage.setItem('todoList', item);
+};
+
+static getLocalStorageData = () => {
+  let todoList;
+
+  if (JSON.parse(localStorage.getItem('todoList')) === null) {
+    todoList = [];
+  } else {
+    todoList = JSON.parse(localStorage.getItem('todoList'));
+  }
+
+  return todoList;
+};
+
+static reassignIndex = (todoList) => {
+  todoList.forEach((item, i) => {
+    item.index = i + 1;
+  });
+}
+
+static deleteTask = (id) => {
+  let todoList = this.getLocalStorageData();
+  const itemToDelete = todoList[id];
+
+  todoList = todoList.filter((item) => item !== itemToDelete);
+
+  this.reassignIndex(todoList);
+  this.setLocalStorageData(todoList);
+};
+
+static updateTaskInput = (newDesc, id) => {
+  const todoList = this.getLocalStorageData();
+  const itemToUpdate = todoList[id];
+
+  todoList.forEach((item) => {
+    if (item === itemToUpdate) {
+      item.description = newDesc;
+    }
+  });
+
+  this.setLocalStorageData(todoList);
+  this.showTodoItems();
+};
+
+static addBtnRemoveEvent = () => {
+  document.querySelectorAll('.trash-can').forEach((button) => button.addEventListener('click', (event) => {
+    event.preventDefault();
+    let id;
+    if (button.id > 0) {
+      id = button.id - 1;
+    } else {
+      id = 0;
+    }
+    this.deleteTask(id);
+    this.showTodoItems();
+  }));
+};
+
+static setUpEdit = (id) => {
+  const todoList = this.getLocalStorageData();
+  const itemToEdit = todoList[id];
+
+  document.getElementById('todo-input').style.display = 'none';
+  const editInput = document.querySelector('.todo-edit-input');
```

Comment:
For JS best practices, it's recommended to use variables in first lines of block scope / global scope 👀 , maybe you can move const editInput to line 68 👍